### DISCUSSION TITLE
【求助】萌新不会配队了！

### DISCUSSION THREAD

**Discussion by SKAIZH** [share121/inter-knot#2819] [其他]
刚入坑十天，目前全部A，S有朱鸢，莱卡恩和格莉丝<img src="https://bbs-static.miyoushe.com/static/2024/03/21/3248c770e7e0e7ec4a167605872b6d66_1511725123905663480.png" alt="丽都漫步-格莉丝" width="75" >

**Comment by SKAIZH** [comment]
可恶啊要不是青衣歪了<img src="https://img-static.mihoyo.com/communityweb/upload/1b5600e9d6d16de86447348909796aa4.png" alt="安比 哭哭" width="75" >

**Comment by WXZ1025** [comment]
警官也逛绳网，真的好吗😱

**Reply by SKAIZH** [reply]
挺好的，黑白通吃

**Comment by I-m-Miss-Lucy-s-dog** [comment]
一隊:朱鳶 安比 妮可 二隊 滿命蒼角 萊卡恩OR派派 露西 如果有手法可以用格莉絲 派派 露西

**Reply by SKAIZH** [reply]
但是我的苍角还是零命(T^T)

**Reply by I-m-Miss-Lucy-s-dog** [reply]
那就派派加露西 记得一号跟三号盘拉高点 派派为了打异常会有扛伤害的需求

**Comment by SKAIZH** [comment]
再帮朋友问问朱鸢 莱卡恩 十一号怎么配

**Reply by I-m-Miss-Lucy-s-dog** [reply]
A级有哪些角色 你去问一下 你朋友 不然不好帮

**Reply by SKAIZH** [reply]
也是都有了

**Reply by I-m-Miss-Lucy-s-dog** [reply]
朱鸢直接照抄我刚刚跟你说的 安比 跟妮可 11号可以配本加上露西 莱卡恩就把苍角命座跟练度拉上組兩人人

**Reply by SKAIZH** [reply]
谢谢(•́⌄•́๑)૭✧
